samples/api/hello_triangle/hello_triangle.cpp

```diff
@@ -543,17 +607,12 @@ void HelloTriangle::init_swapchain()
 	for (size_t i = 0; i < image_count; i++)
 	{
 		// Create an image view which we can render into.
-		VkImageViewCreateInfo view_info{VK_STRUCTURE_TYPE_IMAGE_VIEW_CREATE_INFO};
-		view_info.viewType                    = VK_IMAGE_VIEW_TYPE_2D;
-		view_info.format                      = context.swapchain_dimensions.format;
-		view_info.image                       = swapchain_images[i];
-		view_info.subresourceRange.levelCount = 1;
-		view_info.subresourceRange.layerCount = 1;
-		view_info.subresourceRange.aspectMask = VK_IMAGE_ASPECT_COLOR_BIT;
-		view_info.components.r                = VK_COMPONENT_SWIZZLE_R;
-		view_info.components.g                = VK_COMPONENT_SWIZZLE_G;
-		view_info.components.b                = VK_COMPONENT_SWIZZLE_B;
-		view_info.components.a                = VK_COMPONENT_SWIZZLE_A;
+		VkImageViewCreateInfo view_info{
+		    .sType            = VK_STRUCTURE_TYPE_IMAGE_VIEW_CREATE_INFO,
+		    .image            = swapchain_images[i],
+		    .viewType         = VK_IMAGE_VIEW_TYPE_2D,
+		    .format           = context.swapchain_dimensions.format,
+		    .subresourceRange = {.aspectMask = VK_IMAGE_ASPECT_COLOR_BIT, .baseMipLevel = 0, .levelCount = 1, .baseArrayLayer = 0, .layerCount = 1}};
```

Comment:
It is correct to not explicitly set the `view_info.components`, as they are set to `0` by default, which actually is `VK_COMPONENT_SWIZZLE_IDENTITY`.
But was that skipped intentionally or by accident?

Reply:
Afaik that is correct, and something we did in other samples too.